DependencyInjection/SonataFormatterExtension.php

```diff
@@ -56,8 +56,13 @@ public function load(array $configs, ContainerBuilder $container)
             $loader->load('ckeditor.xml');
         }
 
-        // TODO: To be removed when major version is changed
+        // NEXT_MAJOR: remove this if block
         if (!isset($config['default_formatter'])) {
+            @trigger_error(<<<'EOT'
+Not setting the default_formatter configuration node is deprecated since 3.x,
```

Comment:
@sagikazarmark : I pushed a new version 2bab6fd7786a48ddfbdb44c24c1bec5cf7b56e4d with null logger deprecated, tell me what you think.